src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -1332,6 +1333,128 @@ static int32_t cc_say(InteractiveConsole& console, const utf8** argv, int32_t ar
     }
 }
 
+static int32_t cc_replay_startrecord(InteractiveConsole& console, const utf8** argv, int32_t argc)
+{
+    if (network_get_mode() != NETWORK_MODE_NONE)
+    {
+        console.WriteFormatLine("This command is currently not supported in multiplayer mode.");
+        return 0;
+    }
+
+    if (argc < 1)
+    {
+        console.WriteFormatLine("Parameters required <replay_name> [<max_ticks = 0xFFFFFFFF>]");
+        return 0;
+    }
+
+    std::string name = argv[0];
+
+    // If ticks are specified by user use that otherwise maximum ticks specified by const.
+    uint32_t maxTicks = OpenRCT2::k_MaxReplayTicks;
+    if (argc >= 2)
+    {
+        maxTicks = atol(argv[1]);
+    }
+
+    auto* replayManager = OpenRCT2::GetContext()->GetReplayManager();
+    if (replayManager->StartRecording(name, maxTicks))
+    {
+        console.WriteFormatLine("Replay recording start");
```

Comment:
When recording has started it would be useful to print (both to console and `log_info`) the actual file name used

Reply:
Been wanting to do that but it felt strange to log things twice. I also wonder why we don't print everything to the ingame console already when the UI is up and running. But I can put log_info there.